test/ServerComparison.TestSites.Standalone/project.json

```diff
@@ -1,48 +0,0 @@
-{
-  "version": "1.1.0-*",
-  "dependencies": {
-    "Microsoft.AspNetCore.AspNetCoreModule": "1.0.0-*",
-    "Microsoft.AspNetCore.ResponseCompression": "1.0.0-*",
-    "Microsoft.AspNetCore.Server.HttpSys": "1.2.0-*",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.2.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*",
-    "Microsoft.AspNetCore.WebUtilities": "1.2.0-*",
-    "Microsoft.Extensions.Configuration.CommandLine": "1.2.0-*",
-    "Microsoft.Extensions.Configuration.Json": "1.2.0-*",
-    "Microsoft.Extensions.Logging.Console": "1.2.0-*",
-    "Microsoft.Net.Http.Headers": "1.2.0-*",
-    "Microsoft.NETCore.App": {
-      "version": "1.2.0-*"
-    }
-  },
-  "buildOptions": {
-    "emitEntryPoint": true,
-    "compile": {
-      "include": "../ServerComparison.TestSites/Startup*.cs"
-    }
-  },
-  "publishOptions": {
-    "include": [
-      "web.config"
-    ]
-  },
-  "frameworks": {
-    "netcoreapp1.1": {}
-  },
-  "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
-  },
-  "scripts": {
-    "postpublish": "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%"
-  },
-  "runtimes": {
-    "win7-x64": {},
```

Comment:
We might need to rework how tests handle standalone...per our in-person discussion